cmd/serv.go

```diff
@@ -143,6 +143,12 @@ func runServ(c *cli.Context) error {
 		return nil
 	}
 
+	defer func() {
+		if err := recover(); err != nil {
+			_ = fail(ctx, "Internal Server Error", "Panic: %v", err)
+		}
+	}()
```

Comment:
That's not good.

You hide the panic details, it would cause it becomes very difficult to figure where the panic occurs. You could only see "Panic: out of bound".

Reply:
58fdadd69dd0d1a306c34212a5e1c5966846f871